.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": [
+    "react-app",
+    "airbnb",
+    "plugin:jsx-a11y/recommended",
+    "prettier",
+    "prettier/react"
+  ],
+  "plugins": [
+    "jsx-a11y",
+    "prettier"
+  ],
+  "rules": {
+    "semi": 0,
```

Comment:
can we keep the semi-colon, later we can change of course, just i feel keeping semi-colon make code looks clean.

Reply:
Usually I don't prefer to use semi colons, we can take all others opinions and decide on it.

Reply:
All languages going away from semi colon. Like swift, kotlin and many other programming languages getting rid of semi-colon.

> The fact that there are so many designs, with so much variation, suggest that many language designers view semicolons and braces as syntactic noise to be eliminated if possible. By eliminating syntactic noise, designers make programs easier to read and understand all at once.

https://stackoverflow.com/questions/2848368/why-do-some-languages-not-use-semicolons-and-braces#:~:text=The%20fact%20that%20there%20are,and%20understand%20all%20at%20once.

Reply:
@uday-kybing your's was good reason.